schema.sql

```diff
@@ -0,0 +1,105 @@
+DROP DATABASE readme;
+
+CREATE DATABASE IF NOT EXISTS readme
+  DEFAULT CHARACTER SET utf8
+  DEFAULT COLLATE utf8_general_ci;
+
+USE readme;
+
+CREATE TABLE users (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  email VARCHAR(128) NOT NULL UNIQUE,
+  login VARCHAR(255) NOT NULL UNIQUE,
+  password CHAR(64) NOT NULL,
+  avatar VARCHAR(255) DEFAULT NULL,
+
+  UNIQUE INDEX email_index (email),
+  UNIQUE INDEX login_index (login)
+);
+
+CREATE TABLE types (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  class TINYTEXT DEFAULT NULL
+);
+
+CREATE TABLE posts (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  author INT,
+  type INT,
```

Comment:
Рекомендую 1) не менять название user->author в связях и 2) использовать нотацию имя_связанной_таблицы_id

Именование таблиц в ед. числе желательно для этого, чтобы не получилось `users_id`, что грамматически бессмыслица.

```suggestion
  user_id INT,
  type_id INT,
```